src/collections/_documentation/workflow/releases/index.md

```diff
@@ -353,4 +353,6 @@ You can also use our [API](/api/releases/post-release-deploys/) to create a depl
 
 ## Release Artifacts
 
-JavaScript and iOS projects can utilize release artifacts to unminify or symbolicate error stack traces. To learn more, please check out our [iOS](/clients/cocoa/#sentry-cocoa-debug-symbols) and [JavaScript](/platforms/javascript/sourcemaps/) docs.
+
+JavaScript and React Native projects can utilize release artifacts to unminify or symbolicate error stack traces. You can find the uploaded files in the Settings of your project in the section PROCESSING - Source Maps.
+To learn more, please check out our [JavaScript](/platforms/javascript/sourcemaps/) docs.
```

Comment:
this link is to the Source Maps documentation, so either change the link to the JS index page, or change the link text to indicate the reader will be directed to the Source Maps content.